src/openforms/authentication/contrib/digid_oidc/models.py

```diff
@@ -0,0 +1,201 @@
+    )
+    oidc_rp_client_secret = models.CharField(
+        _("OpenID Connect secret"),
+        max_length=1000,
+        help_text=_("OpenID Connect secret provided by the OIDC Provider"),
+    )
+    oidc_rp_sign_algo = models.CharField(
+        _("OpenID sign algorithm"),
+        max_length=50,
+        help_text=_("Algorithm the Identity Provider uses to sign ID tokens"),
+        default="RS256",
+    )
+    oidc_rp_scopes_list = ArrayField(
+        verbose_name=_("OpenID Connect scopes"),
+        base_field=models.CharField(_("OpenID Connect scope"), max_length=50),
+        default=get_default_scopes,
+        blank=True,
+        help_text=_("OpenID Connect scopes that are requested during login"),
+    )
+
+    oidc_op_discovery_endpoint = models.URLField(
+        _("Discovery endpoint"),
+        max_length=1000,
+        help_text=_(
+            "URL of your OpenID Connect provider discovery endpoint ending with a slash "
+            "(`.well-known/...` will be added automatically). "
+            "If this is provided, the remaining endpoints can be omitted, as "
+            "they will be derived from this endpoint."
+        ),
+        blank=True,
+    )
+    oidc_op_jwks_endpoint = models.URLField(
+        _("JSON Web Key Set endpoint"),
+        max_length=1000,
+        help_text=_(
+            "URL of your OpenID Connect provider JSON Web Key Set endpoint. Required if `RS256` is used as signing algorithm"
+        ),
+        blank=True,
+    )
+    oidc_op_authorization_endpoint = models.URLField(
+        _("Authorization endpoint"),
+        max_length=1000,
+        help_text=_("URL of your OpenID Connect provider authorization endpoint"),
+    )
+    oidc_op_token_endpoint = models.URLField(
+        _("Token endpoint"),
+        max_length=1000,
+        help_text=_("URL of your OpenID Connect provider token endpoint"),
+    )
+    oidc_op_user_endpoint = models.URLField(
+        _("User endpoint"),
+        max_length=1000,
+        help_text=_("URL of your OpenID Connect provider userinfo endpoint"),
+    )
+    oidc_op_logout_endpoint = models.URLField(
+        _("Logout endpoint"),
+        max_length=1000,
+        help_text=_("URL of your OpenID Connect provider logout endpoint"),
+        blank=True,
+    )
+    oidc_rp_idp_sign_key = models.CharField(
+        _("Sign key"),
+        max_length=1000,
+        help_text=_(
+            "Key the Identity Provider uses to sign ID tokens in the case of an RSA sign algorithm. Should be the signing key in PEM or DER format"
+        ),
+        blank=True,
+    )
+
+    oidc_redirect_allowed_hosts = ArrayField(
+        verbose_name=_("Allowed redirect hosts"),
+        base_field=models.CharField(_("Allowed redirect hosts"), max_length=1000),
+        default=list,
+        blank=True,
+        help_text=_(
+            "Hosts that are allowed for redirection after authentication is successful"
+        ),
+    )
```

Comment:
how is this used? how does it compare against our CORS-based redirect validation? I'd like to avoid having to configure the same setting in multiple places & we already perform redirect validation, see https://github.com/open-formulieren/open-forms/blob/master/src/openforms/authentication/views.py#L305

Reply:
I removed this and added the usage `allow_redirect_url` to check this